src/parser/priest/discipline/normalizers/PowerWordRadianceNormalizer.ts

```diff
@@ -37,6 +37,7 @@ class PowerWordRadianceNormalizer extends EventsNormalizer {
         const spellId = event.ability.guid;
         if ((event.timestamp - lastRadianceTimestamp) < MAX_TIME_SINCE_CAST && BUFFS_TO_MOVE.includes(spellId)) {
           event.timestamp = lastRadianceTimestamp;
+          // @ts-ignore
```

Comment:
Instead, because it's actually added in a lot of normalizers, you should add `__modified?: boolean` to the Event interface right below `__fabricated?: boolean`

Reply:
Fixed!